src/index.html

```diff
@@ -13,6 +13,40 @@
     />
```

Comment:
The stylesheet link should reference a CSS file, not a SCSS file. Change 'href="styles/index.scss"' to 'href="styles/index.css"'.